### DISCUSSION TITLE
Other languages besides English?

### DISCUSSION THREAD

**Discussion by rhodilover** [imartinez/privateGPT#866] [Q&A]
Is there a way to be able to interact with privateGPT in other languages?

**Comment by echogit** [comment]
I have the same doubt, in my case, portuguese.
Some other user said it understand arabic but it replies in english, so there is some hope :)

**Reply by AITEK-DEV** [reply]
It's just a suggestion, I haven't delved into this topic yet, but have you tried looking for the languages that the repository contains? If so, why don't you try a newer version of GPT and extract the language and try if PrivateGPT can accept it? It could work, it's the first thing that came to my mind.

**Comment by echogit** [comment]
I already tried to public llama2 "open-source" version and it pretty good.
Trying to set it up locally.

**Reply by essemmeppi** [reply]
Hi echogit – could you tell me how you managed to install llama? is it just downloading it from hugging face and moving it into the "models" folder? i'm trying to build a chatbot for french and it could be very useful. thanks!

**Comment by vilaca** [comment]
is it possible to use privateGPT with PORTULAN/albertina-ptpt?

**Reply by vilaca** [reply]
I've tried PORTULAN/albertina-ptpt for ingestion, results not great

**Reply by nizar0x1f** [reply]
are there any good options?
